### PR TITLE
Add docs about label flag in ex generate, osc new-app, and osc process

### PR DESCRIPTION
**Pull request by 0xmichalis**
Also check in docs for `osc new-app`.

### CODE REVIEW

**Comment by kargakis** [comment]
@bparees ptal

File: using_openshift/templates.adoc [hunk 1]
```diff
@@ -52,7 +52,13 @@ A template describes a set of resources intended to be used together that can be
 ====
 
 == Labels
-link:../architecture/kubernetes_model.html#label[Labels] are used to manage and organize generated resources, such as pods. The labels specified in the template are applied to every resource that is generated from the template.
+link:../architecture/kubernetes_model.html#label[Labels] are used to manage and organize generated resources, such as pods. The labels specified in the template are applied to every resource that is generated from the template. 
+
+There is also the ability to override the labels in the template from the command line.
```
Comment (bparees) [inline using_openshift/templates.adoc]: this isn't an override, you're just indicating another label to be added right?

Comment (kargakis) [inline using_openshift/templates.adoc]: Hmm, I am overriding.

Comment (bparees) [inline using_openshift/templates.adoc]: i'm inclined to say that's not the right thing to do.

Comment (kargakis) [inline using_openshift/templates.adoc]: > i'm inclined to say that's not the right thing to do.

Yea, I agree with you. Will fix it shortly.

**Comment by bparees** [comment]
lgtm.

**Comment by adellape** [comment]
Thanks, merging.

**Pull request merged**